make build-plugin failed

<!--

Welcome to ingress-nginx!  For a smooth issue process, try to answer the following questions.
Don't worry if they're not all applicable; just try to include what you can :-)

If you need to include code snippets or logs, please put them in fenced code
blocks.  If they're super-long, please use the details tag like
<details><summary>super-long log</summary> lots of stuff </details>

-->

<!--

IMPORTANT!!!

Please complete the next sections or the issue will be closed.
This questions are the first thing we need to know to understand the context.

-->

**NGINX Ingress controller version** :

  Release:       v1.0.4
  Build:         git-ed34f6c93

**Kubernetes version**:

none

**Environment**:

- **OS** (e.g. from /etc/os-release):
NAME="Ubuntu"
VERSION="20.04.3 LTS (Focal Fossa)"
- **Kernel** (e.g. `uname -a`):
5.11.0-38-generi

**What happened**:
```
➜  ingress-nginx git:(main) make build-plugin
build/build-plugin.sh: line 75: unexpected EOF while looking for matching `"'
make: *** [Makefile:91: build-plugin] Error 2
```

<!-- (please include exact error messages if you can) -->

**What you expected to happen**:

 cmd `make build-plugin` works correctly
<!-- What do you think went wrong? -->

**Anything else we need to know**:

build-plugin.sh#L60
```
      -X ${PKG}/version.REPO=${REPO_INFO}" \
```
should replace to:
```
     -X ${PKG}/version.REPO=${REPO_INFO} \
```

/kind bug

--- a/build/build-plugin.sh
+++ b/build/build-plugin.sh
@@ -57,7 +57,7 @@ function build_for_arch(){
     -trimpath -ldflags="-buildid= -w -s" \
       -X ${PKG}/version.RELEASE=${TAG} \
       -X ${PKG}/version.COMMIT=${COMMIT_SHA} \
-      -X ${PKG}/version.REPO=${REPO_INFO}" \
+      -X ${PKG}/version.REPO=${REPO_INFO} \
     -o "${release}/kubectl-ingress_nginx${extension}" "${PKG}/cmd/plugin"
 
     cp LICENSE ${release}

